src/state/slices/accountSpecifiersSlice/selectors.ts

```diff
@@ -18,7 +19,7 @@ export const selectAccountSpecifierStrings = (state: ReduxState) =>
 // For most accounts, that's effectively a CAIP19/AssetId, but not for e.g UTXO chains thus the pubkeyish naming
 // We use this in cosmos plugin to get the pubkey as a CAIP19/AssetId, without needing to use chain-adapters in components
 // Since the pubkey is already in state
-export const selectAccountSpecifier = (state: ReduxState, chainId: ChainId) =>
+export const selectAccountSpecifiersByChainId = (state: ReduxState, chainId: ChainId) =>
   state.accountSpecifiers.accountSpecifiers.reduce<string[]>((acc, accountSpecifier) => {
     const pubkeyish = Object.entries(accountSpecifier)[0].join(':')
```

Comment:
I know it wasn't changed in this PR, but it was a recent addition - `pubkeyLike` is probably a more proper name here.

Reply:
We're still having some discussions on the best terminology and `pubKeyish` (or `pubkeyLike` , which would be a very similar one) was a terminology that I actually added in https://github.com/shapeshift/web/commit/b185163e40f7eb954ebdcaaf119fc728db12bb51#diff-bcccd934c3fc099ae907180ecdaa60f38a99865289682fc66896ddba29b6e817R13 and that we're not very happy with yet.
We're still in the process of finding a better naming and consolidate account specifiers / pubkeyishs into a format that will make more sense naming-wise, so I'd rather not change it in this PR until more discussions happen!